app/main/menu.js

```diff
@@ -447,6 +449,23 @@ class AppMenu {
 	static checkForUpdate() {
 		appUpdater(true);
 	}
+
+	static getNextServer(tabs, activeTabIndex) {
```

Comment:
I'm not sure why these methods are required. I guess you're trying to make sure that the user doesn't enter the settings webview - but that doesn't happen anyway. Doesn't the simple `(activeTabIndex + 1) % tabs.length` formula work for this?

Reply:
No, it doesn't. Try opening settings, then go to first org and cycle through using Ctrl+Tab, settings will open up.

Reply:
Ah right, my bad. Awesome, it works as expected. :)